File: plugin/src/org/jetbrains/yesod/hamlet/HamletLanguage.java
package org.jetbrains.yesod.hamlet;

import com.intellij.lang.Language;

public class HamletLanguage extends Language {
    public static final HamletLanguage INSTANCE = new HamletLanguage();

    public HamletLanguage() {
        super("Hamlet", "text/hamlet");
    }
}

File: plugin/src/org/jetbrains/yesod/hamlet/HamletLanguage.kt
package org.jetbrains.yesod.hamlet

/**
 * @author Leyla H
 */

import com.intellij.lang.Language

class HamletLanguage : Language("Hamlet", "text/hamlet") {
    companion object {
        val INSTANCE: HamletLanguage = HamletLanguage()
    }
}